### PR TITLE
cephadm: drop support to python<3.3

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test api

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -1391,32 +1376,10 @@ def call_timeout(ctx, command, timeout):
...
-
-    ret = 1
-    if sys.version_info >= (3, 3):
-        ret = call_timeout_py3(command, timeout)
-    else:
-        # py2 subprocess has no timeout arg
-        ret = call_timeout_py2(command, timeout)
-    return ret
+    try:
+        return subprocess.call(command, timeout=timeout)
```
Comment (sebastian-philipp) [inline src/cephadm/cephadm:1380]: with your changes this should be the same as `call()`, as your call implementation also properly time outs for blocking calls.

Comment (tchaikov) [reply src/cephadm/cephadm:1380]: they are not identical though. the homebrew `call()` also prints to logger. this change aims at removing the python2 support. we can probably switch to `call()` in another PR if you think it justifies the needs, an alternative is to use `subprocess.check_output()`.

**Review by sebastian-philipp — APPROVED** [review]

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2021-02-05_14:59:08-rados-wip-kefu-testing-2021-02-05-1809-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-02-06_12:11:31-rados-wip-kefu-testing-2021-02-06-1838-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-02-07_14:19:29-rados-wip-kefu-testing-2021-02-07-2019-distro-basic-smithi/

reran the failed tests on master

- https://pulpito.ceph.com/kchai-2021-02-08_02:14:21-rados-wip-kefu2-testing-2021-02-08-0023-distro-basic-smithi/

e2e failures tracked by

- https://tracker.ceph.com/issues/49205

**Pull request merged**